packages/inferno-component/src/index.ts

```diff
@@ -19,19 +27,20 @@ import VNodeFlags from 'inferno-vnode-flags';
 let noOp = ERROR_MSG;
 
 if (process.env.NODE_ENV !== 'production') {
-	noOp = 'Inferno Error: Can only update a mounted or mounting component. This usually means you called setState() or forceUpdate() on an unmounted component. This is a no-op.';
+	noOp =
+		'Inferno Error: Can only update a mounted or mounting component. This usually means you called setState() or forceUpdate() on an unmounted component. This is a no-op.';
 }
 
 const componentCallbackQueue: Map<any, Function[]> = new Map();
 
 export interface ComponentLifecycle<P, S> {
-	componentDidMount?(): void;
```

Comment:
@vjeux I'm not convinced we should go with commas for interfaces. Semi-colons are more widespread. I think it makes sense in type literals though.